soql-server-pg/src/test/scala/com/socrata/pg/server/SoQLStringFunctionsTest.scala

```diff
@@ -44,6 +44,7 @@ class SoQLStringFunctionsTest extends SoQLTest {
   }
 
   test("contains(c, x)") {
-    println("Search should be used instead of contains.")
+    // ...What even is this for?
```

Comment:
It is a function to find a substring in a string that has been used in the past to emulate a search.  But it is slow and horrible so we don't support it.  Well... we implement the function just in a way that doesn't actually do contains just equality.  Ideally we would just error out and the test could check that...